batch/batch/worker/worker.py

```diff
@@ -1937,6 +1945,11 @@ async def cleanup(self):
                             f'while unmounting fuse blob storage {bucket} from {mount_path} for job {self.id}'
                         )
 
+        with open('/proc/mounts', 'r') as f:
+            output = f.read()
+            if self.cloudfuse_base_path() in output:
+                raise IncompleteCloudFuseCleanup(f'incomplete cloudfuse unmounting: {output}')
```

Comment:
I'm fairly sure exceptions raised in `DockerJob.cleanup` are not logged anywhere. Using the raise to skip the `xfs_quota` and `rmtree` seems fine but we need to log the exception here.